application/testing/CMakeLists.txt

```diff
@@ -780,6 +780,11 @@ if(VTK_VERSION VERSION_GREATER_EQUAL 9.1.20220519) # Drop file test uses stream
     f3d_test(NAME TestInteractionDropHDRIInvert INTERACTION_CONFIGURE LONG_TIMEOUT)#X;DropEvent palermo.hdr;DropEvent dragon.vtu;
     f3d_test(NAME TestInteractionDropHDRIMulti INTERACTION_CONFIGURE LONG_TIMEOUT)#X;DropEvent dragon.vtu palermo.hdr;
 
+    # Test interactive animation and dropping HDRI
+    if(F3D_SANITIZER STREQUAL "none")
```

Comment:
what's wrong with the sanitizer?

Reply:
same reason as the other tests above:  ` # For some reasons this animation test goes "too far" when run with sanitizer
`